var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -23,6 +23,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from spack import architecture
```

Comment:
Maybe we should just include `architecture` in `__all__` in `lib/spack/spack/__init__.py`.